src/download.rs

```diff
@@ -576,8 +594,15 @@ impl Downloader {
                 None
             }
             Err(e) => {
-                self.fail(anyhow!("Error while downloading media from url {}: {}", task.url, e)).await;
-                None
+                if let GertError::ImgurRemovedError = e {
```

Comment:
nit, we can match directly since we are in a match statement already

``` rust
match result {
            Ok(true) => { //stuff }
            Ok(false) => {
                self.fail(anyhow!("Failed to download media from url: {}", task.url)).await;
                None
            }
            Err(GertError::ImgurRemovedError) {
                 self.skip(format!("Image removed from imgur: {}", task.url)) // already prints
                 None
            },
            Err(e) => {
                self.fail(anyhow!("Error while downloading media from url {}: {}", task.url, e)).await;
                None
            }
        }
```